src/App.tsx

```diff
@@ -25,6 +27,7 @@ export default class App extends React.Component {
             Coveo Headless + Material UI
           </Typography>
           <SearchBox />
+          <Facet title="Facet" />
```

Comment:
Ok so for the past two hours or so, I've been trying to improve the looks of this. Basically, I think that, if we leave this as is, with the facet above the results, it will come off as distracting to readers. We must remember that this is the codesandbox that will appear on all headless pages. So we don't want the facet to take that preeminent a place. 

I haven't quite made it as I would like. I can put the facet to the left of the results, but I would like to have more spacing between rows in the facet and I haven't gotten that to work 😢 Here's a sandbox with what I've got so far: https://codesandbox.io/s/blissful-http-ety21?file=/src/Components/Facet.tsx

I have to work on some other stuff for the rest of the day so I'm leaving this here for now, in case someone else wants to pick it up, and if not then I'll come back to it probably sometime tomorrow :)

Reply:
Thanks for taking the time to do that, JP. The first thing I thought was to maybe change the facet's font colour to a grey that is less prominent than the white so it doesn't compete with the results text.

Reply:
I started from JP's code sandbox and made some additional color, margin, and padding adjustments here:

https://codesandbox.io/s/holy-cdn-635wo?file=/src/Components/Facet.tsx

It's not the prettiest, but I think it does the job of separating the Facet and the ResultList properly.

Reply:
That's awesome, I didn't know that we can customize boxes that elegantly! :D

Reply:
Cool - looks good!

Reply:
I think the changes JP and Kaitlyn did make it look "good enough" that it at least won't be distracting. Good job 👍 

We can/should improve it later. What we had thought of was to collapse the facets under a button (like we do in responsive mode in JSUI).